src/v/kafka/server/quota_manager.cc

```diff
@@ -55,6 +55,10 @@ quota_manager::quota_manager(client_quotas_t& client_quotas)
   , _max_delay(config::shard_local_cfg().max_kafka_throttle_delay_ms.bind()) {
     if (seastar::this_shard_id() == _client_quotas.shard_id()) {
         _gc_timer.set_callback([this]() { gc(); });
+        auto update_quotas = [this]() { update_client_quotas(); };
+        _target_produce_tp_rate_per_client_group.watch(update_quotas);
+        _target_fetch_tp_rate_per_client_group.watch(update_quotas);
+        _target_partition_mutation_quota.watch(update_quotas);
```

Comment:
You're missing `_default_target_produce_tp_rate` and `_default_target_fetch_tp_rate` here

Reply:
for my own benefit, should changes to the defaults necessarily trigger watchers? I could imagine it going either way (i.e. default applies only on bucket creation vs default applies retroactively?)

Reply:
> for my own benefit, should changes to the defaults necessarily trigger watchers? I could imagine it going either way (i.e. default applies only on bucket creation vs default applies retroactively?)

I think either could be argued.